README.md

```diff
@@ -726,13 +726,9 @@ print( // This is probably fine
 C has some built in functions. One of them is the C function. Use it to print the logo of the language.
 
 ```java
-C()!
-/*  ________
-   |  _   __|
-   | | | |
-   | | | |
-   | |_|_|__
-   |________|
+WhenWillProgrammersStopMakingDecisionsForOurSocietyAndJustLeaveUsAloneAlsoHackerNewsIsAVileWebsite()!
+/*  
+WhenWillProgrammersStopMakingDecisionsForOurSocietyAndJustLeaveUsAloneAlsoHackerNewsIsAVileWebsite
```

Comment:
Lovely, but I do prefer the old style. Any chance we can have fully 3D ascii art for the new name?

Reply:
Hmm... Maybe but it will take longer.

Reply:
I made a basic one with only the first letters of the name. (Could you please shorten the name it would help me with bubbly letters).